adapters/axonix/axonix_test.go

```diff
@@ -0,0 +1,30 @@
+package axonix
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamplesWithConfiguredURI(t *testing.T) {
```

Comment:
The /exemplary directory is intended to hold just a few reference examples a publisher can look at for a reference as to how to use your adapter. Additional tests to catch corner cases where there are no errors go into /supplemental. Are you sure all your positive tests belong in /exemplary? too many examples may confuse publishers as to what they should try to follow.